Dangerfile

```diff
@@ -1,5 +1,15 @@
 # frozen_string_literal: true
 
+# Skip Danger run if the opened pull request has "wip" label, [WIP] in the title, or status is Draft
+has_wip_label = github.pr_labels.any? { |label| label.include? 'wip' }
+has_wip_title = github.pr_title.include? '[WIP]'
```

Comment:
```suggestion
has_wip_label = github.pr_labels.any? { |label| label.downcase.include? 'wip' }
has_wip_title = github.pr_title.downcase.include? '[wip]'
```

Should we make it case-insensitive? :)

Reply:
Thanks 🙏🏼 . Fixed in 7e57cc5674f5e5e9916ce4094835242822a179f1